lib/Connection.js

```diff
@@ -281,10 +281,13 @@ class Connection extends BaseConnection {
         if (!this._Ready) return null;
         const [Key, Path] = this._ResolvePath(KeyOrPath.toString());
 
-        let Fetched = this.Cache.has(Key) ? this.Cache.resolve(Key) :
-        JSON.parse(this.API.prepare(`SELECT * FROM '${this.Table}' WHERE Key = ?;`).get(Key)["Val"]);
+        let Fetched = this.Cache.resolve(Key) || (() => {
+            const Req = this.API.prepare(`SELECT * FROM '${this.Table}' WHERE Key = ?;`).get(Key);
+            if (typeof Req !== "undefined") return JSON.parse(Req.Val);
+            else return Req;
+        })();
 
-        if (typeof Fetched === "undefined") return undefined;
+        if (Fetched === null || Fetched === undefined) return Fetched;
```

Comment:
If the row was `null`, it would return null. And if the row didn't exist in the database, it would return `undefined`